capture/build.gradle

```diff
@@ -49,4 +49,8 @@ dependencies {
     implementation libs.androidx.constraintlayout
 
     implementation libs.mavericks
+
+    implementation 'com.google.android.gms:play-services-location:18.0.0'
+    implementation 'androidx.exifinterface:exifinterface:1.3.2'
```

Comment:
Add dependencies to gradle dependency catalog (**libs.versions.toml**)

Reply:
Done